### ISSUE TITLE
Separate functionality to improve reference handling to separate extension or integrate into core?

### ISSUE THREAD

**Issue by sypets** [CodersCare/gridelements#2]
Has integrating this into the core already been suggested / attempted before?

IMHO this functionality has nothing to do with gridelements.

Functionality:

1. offer "Paste as reference" when pasting an element

core:

![image](https://user-images.githubusercontent.com/13206455/181484494-2ebcbaf1-faf5-4e5c-82d4-eeaf89fe14a8.png)

with gridelements:

![image](https://user-images.githubusercontent.com/13206455/181484399-c4d390b9-b756-432f-a3c6-8ca2388a7f02.png)

2. Show content of the source element for references

core:

In the core, you cannot see the content of the original element in the reference (shortcut) - neither the bodytext, nor the image

![image](https://user-images.githubusercontent.com/13206455/181484561-cf49e941-174a-4531-b53d-713490fc917c.png)

gridelements:

In gridelements, this is displayed:

![image](https://user-images.githubusercontent.com/13206455/181485347-7d786ed2-5535-41d5-9a4a-e8ff81ff77ae.png)

**Comment by sypets** [comment]
ok, I was informed there is an extension: https://extensions.typo3.org/extension/paste_reference

**Comment by Cybercraft-GmbH** [comment]
Since this is not really an issue with Gridelements, I will close it here.

The history of Gridelements was always to provide functionality, that is not available in the core yet, but actually should be available. It started with backend layouts after the T3UXW in 2009, which are available in the core now. Actual gridelements, the drag in wizard, the allowed/disallowed/maxitems functionality for colums, the get copy from another page button and of course the reference handling are still missing.

So basically in this case it's the same as with any other functionality Gridelements provides: As soon as this has been implemented in the core, it will be removed from Gridelements.
